src/screens/survey/Question.js

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import { Text, View, Button,StyleSheet,TouchableWithoutFeedback,TouchableHighlight,TouchableOpacity, Image, BackHandler} from 'react-native';
+import {createStackNavigator} from 'react-navigation';
+import Checkbox from './checkbox';
+import { StackActions, NavigationActions } from 'react-navigation';
+import * as questions from './lib/questions.json';
+import * as UserResponse from './lib/userResponses.json'
+
+class Question extends React.Component {
+
+  constructor(props){
+    super(props);
+    this.state = {
+      UserResponse: UserResponse.default,
+      questionStatus: this.props.questionStatus,
+      index: this.props.screenProps[0]
+    }
+    this.nextScreen = this.nextScreen.bind(this);
+    this.changeUserResponseField = this.changeUserResponseField.bind(this);
+  }
+
+  // Sends specific question to All responses made by user
+   sendUserResponse(){
+     this.props.screenProps[2](this.state.index,this.state.UserResponse)
```

Comment:
Also, props are usually named props in a dictionary rather than an array, which makes it a lot simpler to see what you are referring to (instead of screenProps being an array of what looks like [idx, questionName, response?], you could have an api module where you map it to a dict with those names